### PR TITLE
Add lottery announcement

### PR DESCRIPTION
**Pull request by olasitarska**

### CODE REVIEW

File: tinyblog/articles/2016-07-01-registration.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# 2016 Registration
+
+> Published on 1st Jul 2016 by Ola Sitarska
+
+Ever year tickets to Django: Under The Hood are selling out in a matter of minutes.
```
Comment (aniav) [inline tinyblog/articles/2016-07-01-registration.md]: Ever -> Every

File: tinyblog/articles/2016-07-01-registration.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+---
+
+#### The Short Version
+
+1. __On Tuesday, July 12, we're going to open registration.__
+You'll have three weeks to submit your e-mail address for the lottery, until August 2nd at 12 CET.
+
+2. __For three weeks after that, on each Tuesday at 12 CET we're going to randomly 
+select a batch of attendees who will receive a link to purchase their conference tickets.__
+This will happen on July 19th, July 26th and August 2nd. The link to purchase ticket
```
Comment (aniav) [inline tinyblog/articles/2016-07-01-registration.md]: purchase ticket -> ticket purchase

File: tinyblog/articles/2016-07-01-registration.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+That's it! ✨
+
+#### Pricing
+
+For 2016, the full conference passes including 1.5 day of Talks and 1.5 of 
+Sprints will have following pricing:
+
+- 200€ for individual attendees (paying out of their own pocket) 
+- 300€ for corporate tickets (companies buying tickets for their employers)
```
Comment (aniav) [inline tinyblog/articles/2016-07-01-registration.md]: employers -> employees

File: tinyblog/articles/2016-07-01-registration.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+select a batch of attendees who will receive a link to purchase their conference tickets.__
+This will happen on July 19th, July 26th and August 2nd. The link to purchase ticket
+will be valid for 48 hours, and then it will be randomly assigned to another attendee.
+
+That's it! ✨
+
+#### Pricing
+
+For 2016, the full conference passes including 1.5 day of Talks and 1.5 of 
+Sprints will have following pricing:
```
Comment (aniav) [inline tinyblog/articles/2016-07-01-registration.md]: following -> the following

File: tinyblog/articles/2016-07-01-registration.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+would work, but your excitement and care never ceased to surprise us.
+
+The 100 tickets for our very first edition were gone in just few minutes. On our second year,
+we welcomed 3 times as many people, yet the "sold out" banner showed up on our
+tickets page just 10 minutes after registration was open.
+
+The 300 is a number that we're capping the conference at. A sweet spot for serendipity, 
+while still making it possible to organize with our small volunteer team. 
+
+But one thing we didn’t like about the registration proccess in the last two years — the
```
Comment (aniav) [inline tinyblog/articles/2016-07-01-registration.md]: There should be no spaces around "—"

**Comment by aniav** [comment]
Just a few nitpicks. Otherwise 💃 💃 💃 from me :D

File: tinyblog/articles/2016-07-01-registration.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+a little bit more fair and less frustrating for everyone.
+
+Read on to learn about the important changes we're making this year!
+
+---
+
+#### The Short Version
+
+1. __On Tuesday, July 12, we're going to open registration.__
+You'll have three weeks to submit your e-mail address for the lottery, until August 2nd at 12 CET.
```
Comment (asendecka) [inline tinyblog/articles/2016-07-01-registration.md]: Is it noon or midnight? 🕚

File: tinyblog/articles/2016-07-01-registration.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+2. __For three weeks after that, on each Tuesday at 12 CET we're going to randomly
```
Comment (asendecka) [inline tinyblog/articles/2016-07-01-registration.md]: Once again: we should specify if it's am or pm.

Comment (erikr) [inline tinyblog/articles/2016-07-01-registration.md]: Or: noon / midnight. I never know which one is 12 AM and 12 PM.

**Comment by erikr** [comment]
For the record, here's the alternate procedure I just proposed on Slack:

1. At time X, we draw ~200 winners from the applicants.
2. All winners have 72 hours to register their ticket.
3. After 72 hours, we re-draw for however many tickets are unclaimed. Previous winners (whether or not they registered) are excluded from this drawing.
4. The second set of winners is also given 72 hours.
5. Repeat from step 3

File: templates/_tickets.html [hunk 8]
```diff
@@ -9,16 +9,8 @@
...
-                    <span data-emoji-alt="🎫">Tickets</span> will go on sale in:
-                    <strong>
-                        <span class="countdown-container"  title="{{ ticket_sale_start.isoformat }}">
-                            {{ ticket_sale_start|timeuntil }}
-                        </span>
-                    </strong>
-                </p>
-                {% endcomment %}
+                
+                <p>🎫 <a href="/blog/article/2016-07-01-registration/">Read on our blog how are we planning to sell conference tickets this year</a></p>
```
Comment (mjtamlyn) [inline templates/_tickets.html]: How we plan

File: tinyblog/articles/2016-07-01-registration.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+This year we're trying something different to make the distribution of tickets
+a little bit more fair and less frustrating for everyone.
+
+Read on to learn about the important changes we're making this year!
+
+---
+
+#### The Short Version
+
+1. __On Tuesday, July 12, we're going to open registration.__
```
Comment (mjtamlyn) [inline tinyblog/articles/2016-07-01-registration.md]: we will open

File: tinyblog/articles/2016-07-01-registration.md [hunk 10]
```diff
@@ -0,0 +1,72 @@
+# 2016 Registration
+
+> Published on 1st Jul 2016 by Ola Sitarska
+
+Every year tickets to Django: Under The Hood are selling out in a matter of minutes.
```
Comment (mjtamlyn) [inline tinyblog/articles/2016-07-01-registration.md]: tickets ... sell out in

File: tinyblog/articles/2016-07-01-registration.md [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+we'd be happy to discuss a sponsorship opportunity with you. Please get in touch
+with us on [hello@djangounderthehood.com](mailto:hello@djangounderthehood.com).
+
+#### The Slightly Longer Version
+
+Django: Under The Hood started as this little thing for around 100 people who joined
+us in Amsterdam for the first time about 2.5 years ago. We had no idea if this 
+would work, but your excitement and care never ceased to surprise us.
+
+The 100 tickets for our very first edition were gone in just few minutes. On our second year,
```
Comment (mjtamlyn) [inline tinyblog/articles/2016-07-01-registration.md]: In our second year

File: tinyblog/articles/2016-07-01-registration.md [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+we welcomed 3 times as many people, yet the "sold out" banner showed up on our
+tickets page just 10 minutes after registration was open.
+
+The 300 is a number that we're capping the conference at. A sweet spot for serendipity,
```
Comment (mjtamlyn) [inline tinyblog/articles/2016-07-01-registration.md]: We are capping the conference at 300 attendees. It's a good balance between having as many attendees as possible and making it possible to organise with our small volunteer team.

File: tinyblog/articles/2016-07-01-registration.md [hunk 13]
```diff
@@ -0,0 +1,72 @@
...
+a little bit more fair and less frustrating for everyone.
+
+Read on to learn about the important changes we're making this year!
+
+---
+
+#### The Short Version
+
+1. __On Tuesday, July 12, we're going to open registration.__
+You'll have two weeks to submit your e-mail address for the lottery, until July 19th at noon CET.
```
Comment (oinopion) [inline tinyblog/articles/2016-07-01-registration.md]: I find it useful to like times to `time.is`: http://time.is/1200_19_July_2016_in_Amsterdam

File: tinyblog/articles/2016-07-01-registration.md [hunk 14]
```diff
@@ -0,0 +1,72 @@
...
+would work, but your excitement and care never ceased to surprise us.
+
+The 100 tickets for our very first edition were gone in just few minutes. On our second year,
+we welcomed 3 times as many people, yet the "sold out" banner showed up on our
+tickets page just 10 minutes after registration was open.
+
+The 300 is a number that we're capping the conference at. A sweet spot for serendipity, 
+while still making it possible to organize with our small volunteer team. 
+
+But one thing we didn’t like about the registration process in the last two years: the
```
Comment (mjtamlyn) [inline tinyblog/articles/2016-07-01-registration.md]: However, one thing ... years was the feeling

File: tinyblog/articles/2016-07-01-registration.md [hunk 15]
```diff
@@ -0,0 +1,72 @@
...
+
+The 100 tickets for our very first edition were gone in just few minutes. On our second year,
+we welcomed 3 times as many people, yet the "sold out" banner showed up on our
+tickets page just 10 minutes after registration was open.
+
+The 300 is a number that we're capping the conference at. A sweet spot for serendipity, 
+while still making it possible to organize with our small volunteer team. 
+
+But one thing we didn’t like about the registration process in the last two years: the 
+feeling of panic and chaos that came from that confusion, and the subsequent mad rush
```
Comment (mjtamlyn) [inline tinyblog/articles/2016-07-01-registration.md]: came from the mad rush

**Pull request merged**